packages/pact-config/src/__tests__/setup.pact.test.ts

```diff
@@ -1,15 +1,31 @@
 import {Interaction, Matchers} from "@pact-foundation/pact"
-import {pactSetup} from "../index"
-
+import {pactSetup, pactsOutDir} from "../index"
 import {MenuService} from "../__mocks__/menuService"
+import {unlink, access, constants} from "fs"
 
-describe("Yumido Menu API", () => {
-    const url = "http://localhost"
-    const port = 8257
+process.env.PACT_CONSUMER = "test_consumer"
```

Comment:
What is the difference between setting it here and setting the env vars in the package script? I mean which is better?

Reply:
I wanted this in context of the tests, but good shout.